src/main/java/uk/gov/hmcts/darts/common/client/exeption/AbstractClientProblemDecoder.java

```diff
@@ -35,11 +40,12 @@ public Exception decode(String methodKey, Response response) {
     }
 
     public DartsException decode(HttpStatusCodeException response) {
+        log.trace(RESPONSE_PREFIX.concat("{}"), response.getMessage());
         try (ByteArrayInputStream is = new ByteArrayInputStream(response.getResponseBodyAsByteArray())) {
             Problem problem = getProblem(is);
             return getExceptionForProblem(problem);
         } catch (Exception ioEx) {
-            log.error("Failed to read the problem json", ioEx);
+            log.error("Failed to read the problem json so");
```

Comment:
why do we have so at the end?

Should also include the exception in the log for easier debugging

Reply:
Done